app/scripts/controllers/transactions/index.js

```diff
@@ -1512,12 +1525,15 @@ export default class TransactionController extends EventEmitter {
 
     const gasParamsInGwei = this._getGasValuesInGWEI(gasParams);
 
+    const { eip1559V2Enabled: eip1559V2 } = this.preferencesStore.getState();
+
     const properties = {
       chain_id: chainId,
       referrer,
       source,
       network,
       type,
+      eip1559V2,
```

Comment:
[REQ]: `eip_1559_version: eip1559V2 ? '2' : '1';` (opens up better options after v3 goes live.

Reply:
I just updated the PR with change.